deps/v8/gypfiles/features.gypi

```diff
@@ -103,7 +103,8 @@
     # Enable mitigations for executing untrusted code.
     'v8_untrusted_code_mitigations%': 'true',
 
-    'v8_enable_handle_zapping%': 1,
+    # Should be turned on only for debugging
```

Comment:
… which is not fixed yet in this PR, according to https://ci.nodejs.org/job/node-test-commit-linux-containered/8002/nodes=ubuntu1604_sharedlibs_debug_x64/console?

Reply:
Ack